source/uwp/Visualizer/CustomElements/CustomInput.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using AdaptiveCards.Rendering.Uwp;
+using Newtonsoft.Json;
+using Windows.Data.Json;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace XamlCardVisualizer.CustomElements
+{
+    public class CustomInput : IAdaptiveInputElement, IAdaptiveCardElement
+    {
+        public string ErrorMessage { get; set; }
+        public bool IsRequired { get; set; }
+        public string Label { get; set; }
+
+        public static readonly string customInputType = "customInput";
+
+        public JsonObject ToJson()
+        {
+            return null;
+        }
+
+        public JsonObject AdditionalProperties { get; set; }
+
+        public ElementType ElementType => ElementType.Custom;
+
+        public string ElementTypeString => customInputType;
+
+        public IAdaptiveCardElement FallbackContent { get; set; }
+        public FallbackType FallbackType { get; set; }
+        public HeightType Height { get; set; }
+        public string Id { get; set; }
+        public bool IsVisible { get; set; } = true;
+
+        public IList<AdaptiveRequirement> Requirements = new List<AdaptiveRequirement>();
+
+        public bool Separator { get; set; }
+        public Spacing Spacing { get; set; }
+
+        IList<AdaptiveRequirement> IAdaptiveCardElement.Requirements => throw new NotImplementedException();
+    }
+
+
+    public class CustomInputParser : IAdaptiveElementParser
+    {
+        public IAdaptiveCardElement FromJson(JsonObject inputJson, AdaptiveElementParserRegistration elementParsers, AdaptiveActionParserRegistration actionParsers, IList<AdaptiveWarning> warnings)
+        {
+            CustomInput customInput = new CustomInput();
+            customInput.Id = "id1";
```

Comment:
I know we're avoiding having to write a full parser here, but consider using a random `Id` here (maybe a GUID?) just so we don't have to worry about future test cards using more than one `CustomInput` :)